homeassistant/components/fan/significant_change.py

```diff
@@ -0,0 +1,61 @@
+"""Helper to test significant Fan state changes."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.significant_change import check_absolute_change
+
+from . import ATTR_PERCENTAGE, ATTR_PERCENTAGE_STEP
+
+INSIGNIFICANT_ATTRIBUTES: set[str] = {ATTR_PERCENTAGE_STEP}
+
+
+def _check_valid_float(value: str | int | float) -> bool:
```

Comment:
i think this could also be a function in `homeassistant.helpers.significant_change`

Reply:
#106005